--- man-pages-ja-20060815.orig/manual/GNU_groff/translation_list	2006-08-15 00:07:02.000000000 +0900
+++ man-pages-ja-20060815/manual/GNU_groff/translation_list	2009-11-18 17:07:09.000000000 +0900
@@ -29,7 +29,6 @@
 £Ã:GNU groff:1.17.2:2000/05/24:groff:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
 £Ã:GNU groff:1.17.2:2000/05/19:groff_char:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
 £Ã:GNU groff:1.17.2:2000/05/22:groff_man:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
-£Ã:GNU groff:1.17.2:1997/11/24:mdoc:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
 £Ã:GNU groff:1.17.2:2000/02/06:groff_me:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
 £Ã:GNU groff:1.17.2:2000/06/05:groff_mm:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
 £Ã:GNU groff:1.17.2:2000/02/22:groff_mmse:7:2002/01/05::man-jp@jp.FreeBSD.org:FreeBSD jpman project:http://www.jp.freebsd.org/man-jp/
